skrub/selectors/__init__.py

```diff
@@ -1,240 +1,3 @@
-"""
```

Comment:
I was about to merge (I did a small change), and then I realized that this large section has disappeared.

I wanted to check that it is ok

Reply:
Yes, I'd say 80% of it has been pushed to the User guide, and the remaining 20% was mostly internal dev stuff. I brought these 20% back to this file, with a link to the user guide for the public-facing part of the selector API.